packages/ckeditor5-dev-env/lib/translations/collect-utils.js

```diff
@@ -0,0 +1,202 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const glob = require( 'glob' );
+const fs = require( 'fs-extra' );
+const path = require( 'path' );
+const logger = require( '@ckeditor/ckeditor5-dev-utils' ).logger();
+
+const ckeditor5PackagesDir = path.join( process.cwd(), 'packages' );
+const langContextSuffix = path.join( 'lang', 'contexts.json' );
+const corePackageName = 'ckeditor5-core';
+
+const utils = {
+	/**
+	 * Collect translations and returns array of translations.
+	 *
+	 * @returns {Array<Object>}
```

Comment:
This is a correct format, but we use its alternative, with a period after `Array`: `Array.<Object>`.